jax/experimental/README.md

```diff
@@ -7,6 +7,10 @@ hundred lines of code, so take a look inside and adapt them as you need!
 
 ### Neural-net building with Stax
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
```

Comment:
Perhaps limit this text to apply just to Stax for now? It's not clear what the policy is for other libraries in `experimental`.

Reply:
SGTM! Made a modification to refer to only Stax.